index.html

```diff
@@ -17,13 +17,82 @@
     <link rel="stylesheet" href="./css/styles.css" />
     <title>Britlex</title>
   </head>
-    <body>
+  <body>
     <div class="wrapper">
-<header></header>
-<main>
-  <section></section>
-</main>
-<footer></footer>
+      <header></header>
+      <main>
+        <section class="container skills">
+          <h2 class="section-title">Skills</h2>
+          <ul class="skills-list">
```

Comment:
Add rounding to each child of skills-list

Reply:
done

Reply:
The blocks still have right angles